src/libNode/Node.cpp

```diff
@@ -3224,4 +3224,12 @@ void Node::CleanLocalRawStores() {
       }
     }
   }
+}
+void Node ::storeVoteUntilPow(const std::string& proposalId,
+                              const std::string& vote) {
+  m_govProposal = make_pair(static_cast<uint32_t>(std::stoul(proposalId)),
```

Comment:
Can this potentially throw an uncaught exception?

Reply:
yeah.correct. Fixing it. `stoul` can throw exception.Thanks.